src/core/server/integration_tests/ci_checks/saved_objects/check_registered_types.test.ts

```diff
@@ -46,6 +47,8 @@ describe('checking migration metadata changes on all registered SO types', () =>
   // This test is meant to fail when any change is made in registered types that could potentially impact the SO migration.
   // Just update the snapshot by running this test file via jest_integration with `-u` and push the update.
   // The intent is to trigger a code review from the Core team to review the SO type changes.
+  // The number of types in the hashMap should never be reduced, it can only increase.
+  // Removing saved object types is forbidden after 8.8.
```

Comment:
These comments are ment to warn against removing saved object types for anyone who ends up having to make changes to the tests to get them to pass.